AMEBI.Mvc/Controllers/AccountController.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Threading.Tasks;
+using AMEBI.Domain.Services;
+using AMEBI.Domain.ViewModel;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AMEBI.Mvc.Controllers
+{
+    [Route("[controller]")]
+    public class AccountController : Controller
+    {
+        private readonly IUserService _userService;
+        private readonly IJwtService _jwt;
+
+        public AccountController(IUserService userService, IJwtService jwt)
+        {
+            _userService = userService;
+            _jwt = jwt;
+        }
+        
+        [HttpPost]
+        [AllowAnonymous]
+        [Route("login")]
+        public async Task<IActionResult> Login([FromBody]LoginViewModel model)
+        {   
+            var user = await _userService.FindAsync(model.Username);
```

Comment:
Is it really necessary to use DbContext for operating on users? I think we should only use LDAP.

Reply:
Like comment about LDAP above...